edk2toolext/invocables/edk2_setup.py

```diff
@@ -173,10 +194,13 @@ def Go(self):
                     # If we're not skipping, grab it.
                     if not skip_repo or self.force_it:
                         logging.info("## Fetching repo.")
-                        cmd_string = "submodule update --init --recursive --progress"
+                        cmd_string = "submodule update --init"
```

Comment:
A pattern that we use is creating a list and appending to the list conditionally and then joining it later. Not sure if you want to use that here.

Reply:
i don't think it is worth it here.